app/assets/stylesheets/mixins/course/forum/_forum_post.scss

```diff
@@ -0,0 +1,34 @@
+$border_color: #ddd;
+$shadow_color: rgba(0, 0, 0, 0.075);
```

Comment:
Name of variable `shadow_color` should be written in all lowercase letters with hyphens instead of underscores